pkg/generate/app/cmd/newapp.go

```diff
@@ -386,6 +425,19 @@ func (c *AppConfig) Run(out io.Writer) (*AppResult, error) {
 
 	objects = app.AddServices(objects)
 
+	templates, err := c.buildTemplates(components, app.Environment(parameters))
+	if err != nil {
+		return nil, err
+	}
+	for _, t := range templates {
+		templateResolver := c.templateResolver.(app.TemplateResolver)
+		items, err := t.Process(templateResolver.TemplateConfigsNamespacer, templateResolver.Namespaces[0])
```

Comment:
Is it ok to take the client from the resolver or should I just store it in `AppConfig`? I need it to be able to `Process`.